pages/common/git-mv.md

```diff
@@ -0,0 +1,11 @@
+# git mv
+
+> Move or rename files and update the git index.
+
+- Rename or move file and add it to next commit
```

Comment:
I'd make separate examples for moving and renaming, with the moving example including a relative path in the filename parameter (and no "new" prefix), to make the distinction clearer.